ts/sagas/startup.ts

```diff
@@ -512,38 +506,23 @@ export function* initializeApplicationSaga(): Generator<
   // Load visible services and service details from backend when requested
   yield* fork(watchLoadServicesSaga, backendClient);
 
-  // Load all messages when requested
-  yield* fork(watchLoadMessages, backendClient.getMessages);
-
-  if (usePaginatedMessages) {
-    yield* fork(watchLoadNextPageMessages, backendClient.getMessages);
-    yield* fork(watchLoadPreviousPageMessages, backendClient.getMessages);
-    yield* fork(watchReloadAllMessages, backendClient.getMessages);
-    yield* fork(watchLoadMessageById, backendClient.getMessage);
-    yield* fork(watchLoadMessageDetails, backendClient.getMessage);
-    yield* fork(
-      watchUpsertMessageStatusAttribues,
-      backendClient.upsertMessageStatusAttributes
-    );
-    yield* fork(
-      watchMigrateToPagination,
-      backendClient.upsertMessageStatusAttributes
-    );
-  }
+  yield* fork(watchLoadNextPageMessages, backendClient.getMessages);
+  yield* fork(watchLoadPreviousPageMessages, backendClient.getMessages);
+  yield* fork(watchReloadAllMessages, backendClient.getMessages);
+  yield* fork(watchLoadMessageById, backendClient.getMessage);
+  yield* fork(watchLoadMessageDetails, backendClient.getMessage);
+  yield* fork(
+    watchUpsertMessageStatusAttribues,
+    backendClient.upsertMessageStatusAttributes
+  );
+  yield* fork(
+    watchMigrateToPagination,
```

Comment:
Is this still needed?

Reply:
@francescopersico yes, if you have installed a version before pagination and update to the latest one, your messages need to be migrated (i.e. upsert archived and read state of your messages to the backend).

Reply:
Normal state migration is not enough right?